### PR TITLE
Fixed incorrect docstrings in rosnode/__init__.py

### PR DESCRIPTION
**Pull request by ajeetdsouza**
Relevant issue: #1254 
I closed the previous pull request, since it had conflicts.

### CODE REVIEW

File: tools/rosbag/src/rosbag/bag.py [hunk 1]
```diff
@@ -778,7 +778,7 @@ def __str__(self):
                     else:
                         rows.append(('', s))
         
-        except Exception as ex:
+        except:
```
Comment (dirk-thomas) [inline tools/rosbag/src/rosbag/bag.py]: Please do not remove the `Exception` type. This changes behavior and catches other exception not subclasses from `Exception` like `KeyboardInterrupt`.

Same below.

**Review by dirk-thomas — COMMENTED** [review]
Please do not mix unrelated changes in a single PR. The title of this PR is about fixing docstring.

Please create separate patches for removing the unnecessary variables.

**Comment by ajeetdsouza** [comment]
Sorry! I've reverted the unrelated commits, could you check?

**Comment by dirk-thomas** [comment]
Thank you for the patch.

**Pull request merged**